.github/workflows/release-notes.yaml

```
@@ -0,0 +1,76 @@
+          fi
+          VERSION_WITH_PREFIX=$(echo "$RELEASE_INFO" | jq -r .tag_name)
+          VERSION=$(echo "$VERSION_WITH_PREFIX" | sed 's/^[vV]//')
+          NOTES=$(echo "$RELEASE_INFO" | jq -r .body)
+          echo "VERSION_WITH_PREFIX=${VERSION_WITH_PREFIX}" >> $GITHUB_ENV
+          echo "VERSION=${VERSION}" >> $GITHUB_ENV
+          echo 'NOTES<<EOF' >> $GITHUB_ENV
+          echo "$NOTES" >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+
+      - name: Append release notes to changelog.md
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2181:style:3:6: Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?

(shellcheck)